tasks.py

```diff
@@ -61,10 +61,14 @@ def install(c):
     """
     Install the application (composer, yarn, ...)
     """
-    # with Builder(c):
-    #     docker_compose_run(c, 'composer install -n --prefer-dist --optimize-autoloader', no_deps=True)
-    #     run_in_docker_or_locally_for_dinghy(c, 'yarn', no_deps=True)
-
+    with Builder(c):
+        if os.path.isfile('composer.json'):
+            docker_compose_run(c, 'composer install -n --prefer-dist --optimize-autoloader', no_deps=True)
+        if os.path.isfile('yarn.lock'):
+            run_in_docker_or_locally_for_dinghy(c, 'yarn', no_deps=True)
+        if os.path.isfile('package.json'):
```

Comment:
I'm afraid `os.path.isfile` is run on the host, so the path is relative to this tasks.py. 

As the default directory structure is to have the application files in a **application** subdirectory this will not work as expected.

We need to use `c.root_dir + c.project_directory` to check files in the working directory of the builder.

Reply:
corrected ! thanx for this great kit !